tempesta_fw/http2/rotate.h

```diff
@@ -0,0 +1,266 @@
+/**
+ *		Tempesta FW
+ *
+ * Conversion between little and big endian numbers.
```

Comment:
The comment is just the same as for netconv.h, so is it just one more duplication of a code available in Linux?